stdlib/3.5/pathlib.pyi

```diff
@@ -1,33 +1,32 @@
-# Stubs for pathlib (Python 3.4)
+# Stubs for pathlib (Python 3.5)
 
-from typing import Any, Generator, IO, Optional, Sequence, Tuple, Union
+from typing import Any, AnyStr, Generator, IO, Optional, Sequence, Tuple, Union
 import os
 
-class PurePath:
+class PurePath(object):
     parts = ...  # type: Tuple[str, ...]
-    drive = ... # type: str
-    root = ... # type: str
+    drive = ...  # type: str
+    root = ...  # type: str
     anchor = ...  # type: str
     parents = ...  # type: Sequence[PurePath]
     parent = ...  # type: PurePath
     name = ...  # type: str
     suffix = ...  # type: str
     suffixes = ...  # type: List[str]
     stem = ...  # type: str
-    def __init__(self, *pathsegments: str) -> None: ...
-    def __hash__(self) -> int: ...
```

Comment:
I removed it because it exists on `object` and this doesn't change the signature, so it seemed unnecessary.